kustomization.yaml

```diff
@@ -3,4 +3,4 @@ resources:
 
 images:
 - name: ghcr.io/cybozu-go/moco
-  newTag: 0.20.2
```

Comment:
We should not update here until MOCO v0.20.3 is released.

Reply:
I revert it to 0.20.2